GETTING_STARTED.md

```diff
@@ -41,6 +41,7 @@ class Test::Unit::TestCase
 end
 
 # Cucumber
+# env.rb (Rails example location - app/features/support/env.rb)
```

Comment:
including the `app/` prefix here is a bit confusing to me, since that's where models, controllers, etc. live. How do you feel about `RAILS_ROOT` instead?